crates/pathfinder/fixtures/sequencer/valid_tx_status.json

```diff
@@ -0,0 +1,4 @@
+{
+    "tx_status": "ACCEPTED_ON_L1",
+    "block_hash": "0x9aaa1733f916339979d0df10e2969c4a12146e80c8aa5bafbec876605bf35a"
+}
```

Comment:
Would fix these fixtures as well with newline at the end to make them text files, as they are even formatted..

Reply:
I think let's just agree on one convention for the fixtures:
- formatted, with newline at the end of file, or
- raw, as in #308

Reply:
I've also commented on those "raw" fixtures, I think the root cause for those was just misconfigured IDE. The convention of having newlines at the end of text files is quite old, and I don't see a point in creating new one convention.

Reply:
Ok, so formatted with newlines at the eof.

Reply:
> I've also commented on those "raw" fixtures, I think the root cause for those was just misconfigured IDE.

No, not really (that would have been the case for formatted JSONs). I'm OK with formatted JSONs with newlines, though, will update #308 too once I get back to block hashes.